### DISCUSSION TITLE
Can someone please explain to me why the Process Image is changing image during resizing process? Isn't the image simply supposed to be upscaled without any changes to the characteristics or quality?

### DISCUSSION THREAD

**Discussion by superprohero** [vladmandic/sdnext#859] [Q&A]
before resizing
![hhh](https://github.com/vladmandic/automatic/assets/133063465/d7266805-c89e-4caa-b2c8-3739041a558a)

after resizing
![hjkhjk](https://github.com/vladmandic/automatic/assets/133063465/5c76104b-c653-4df9-8c21-c44135c73398)

It did make the image better, but I think the resolution dropped after resizing for some reason.

Upscaler I used SwinIR 4x
![sgjhsdghsdhuo](https://github.com/vladmandic/automatic/assets/133063465/3e1466bf-72f8-4a77-8c98-3f8458b24a27)

MY UI image

**Comment by superprohero** [comment]
I solved the problem, Set CodeFormer visibility and weight to 0 to not change faces.

**Reply by brknsoul** [reply]
You want CodeFormer Visibility at 0, and Weight at 1.
For some strange reason Weight = 1.0 is minimum effect.
